### ISSUE TITLE
hope-tab text color should be set dynamically based on theme-color

### ISSUE THREAD

**Issue by wilsonpage** [etiennesegonzac/APZ-gambler#25]
A light theme color should have dark text and a dark theme color should have light text.

**Comment by wilsonpage** [comment]
[Here's a lightness test](http://jsbin.com/necucar/edit?js,console,output). I don't know enough about colors to know if it's good or not.

**Comment by wilsonpage** [comment]
It seems to declare red as 'light', but per our design I think we regard that as 'dark'. So it may need some tweaking.

**Comment by jonathanKingston** [comment]
The algo defines for CSS `contrast()` might be worth looking at as an alternative here: https://drafts.csswg.org/css-color-4/#contrast-adjuster

I suspect that won't give as nice shades as just white and black text though but interesting prior art all the same.

**Comment by jonathanKingston** [comment]
Also this is interesting and relevant: https://blog.mozilla.org/ux/2015/12/firefox-os-customization-concept/
(However the text color doesn't seem to change)

**Comment by etiennesegonzac** [comment]
@jonathanKingston yep, Peter La is also working on this project so it might very well happen :)

**Comment by pivanov** [comment]
Hey guys,
If you need some help about tweaking colors just ping me :)

**Comment by etiennesegonzac** [comment]
@pivanov  Feel free to give it a try! And let me know if there's anything blocking.
